syntax = "proto3";

package calculator;

message PrimeDecompositionRequest {
  uint32 number = 1;
}

message PrimeDecompositionResponse {
  uint32 prime = 1;
}